core/src/operations/mod.rs

```diff
@@ -387,105 +392,107 @@ impl Operation {
-            Self::Dup11 => Some(0b0001_1001),
-            Self::Dup13 => Some(0b0001_1011),
-            Self::Dup15 => Some(0b0001_1100),
-
-            Self::Swap => Some(0b0010_0000),
-            Self::SwapW => Some(0b0010_0001),
-            Self::SwapW2 => Some(0b0010_0010),
-            Self::SwapW3 => Some(0b0010_0011),
-
-            Self::MovUp2 => Some(0b0010_0001),
-            Self::MovUp3 => Some(0b0010_0010),
-            Self::MovUp4 => Some(0b0010_0011),
-            Self::MovUp5 => Some(0b0010_0100),
-            Self::MovUp6 => Some(0b0010_0101),
-            Self::MovUp7 => Some(0b0010_0110),
-            Self::MovUp9 => Some(0b0010_0111),
-            Self::MovUp11 => Some(0b0010_1000),
-            Self::MovUp13 => Some(0b0010_1001),
-            Self::MovUp15 => Some(0b0010_1011),
-
-            Self::MovDn2 => Some(0b0010_0110),
-            Self::MovDn3 => Some(0b0010_0111),
-            Self::MovDn4 => Some(0b0010_1000),
-            Self::MovDn5 => Some(0b0010_1001),
-            Self::MovDn6 => Some(0b0010_1010),
-            Self::MovDn7 => Some(0b0010_1010),
-            Self::MovDn9 => Some(0b0010_1010),
-            Self::MovDn11 => Some(0b0010_1010),
-            Self::MovDn13 => Some(0b0010_1010),
-            Self::MovDn15 => Some(0b0010_1010),
-
-            Self::CSwap => Some(0b0010_1010),
-            Self::CSwapW => Some(0b0010_1010),
-
-            Self::U32split => Some(0b0011_0000),
-            Self::U32add => Some(0b0011_0001),
-            Self::U32addc => Some(0b0011_0010),
-            Self::U32sub => Some(0b0011_0011),
-            Self::U32mul => Some(0b0011_0100),
-            Self::U32madd => Some(0b0011_0101),
-            Self::U32div => Some(0b0011_0110),
-            Self::U32and => Some(0b0011_0111),
-            Self::U32or => Some(0b0011_1000),
-            Self::U32xor => Some(0b0011_1001),
-            Self::U32assert2 => Some(0b0111_1001),
-
-            Self::LoadW => Some(0b0011_1010),
-            Self::StoreW => Some(0b0011_1011),
-
-            Self::Read => Some(0b0011_1100),
-            Self::ReadW => Some(0b0011_1101),
-
-            Self::SDepth => Some(0b0011_1101),
-
-            Self::RpPerm => Some(0b0011_1111),
-            Self::MpVerify => Some(0b0011_1110),
-            Self::MrUpdate(_) => Some(0b0011_1110),
-
-            Self::End => Some(0b0111_0000),
-            Self::Join => Some(0b0111_0001),
-            Self::Split => Some(0b0111_0010),
-            Self::Loop => Some(0b0111_0011),
-            Self::Repeat => Some(0b0111_0100),
-            Self::Respan => Some(0b0111_1000),
-            Self::Span => Some(0b0111_1111),
+            Self::Noop => Some(0),
+            Self::Assert => Some(1),
+
+            Self::FmpAdd => Some(2),
+            Self::FmpUpdate => Some(3),
+
+            Self::Push(_) => Some(4),
+
+            Self::Eq => Some(5),
+            Self::Eqz => Some(6),
+            Self::Eqw => Some(7),
```

Comment:
Just to clarify - are these changes just to ensure uniqueness until #203 is tackled?

Reply:
Yes - correct!